tests/dummy/app/pods/toggle/template.hbs

```diff
@@ -0,0 +1,104 @@
+<div class='dummy-notification-container'>
+  {{#each notifications as |notification|}}
+    {{notification-message notification=notification}}
+  {{/each}}
+</div>
+
+<div class='dummy-body button'>
+    <div class='section-title'>Toggle button</div>
+    <hr>
+    <div class='section'>
+        <div class='example'>
+            <div class='title'>action syntax</div>
+            <div class='demo'>
+              {{! BEGIN-SNIPPET button-primary-small }}
```

Comment:
Use `toggle` instead of `button` here or it won't be unique in the future - also I don't think `primary-small` is the case being shown here - it's the action right?  So this should be `toggle-events`